autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -204,24 +204,43 @@ def fit(train_data,
                 Options include: 'NN' (neural network), 'GBM' (lightGBM boosted trees), 'CAT' (CatBoost boosted trees), 'RF' (random forest), 'XT' (extremely randomized trees), 'KNN' (k-nearest neighbors)
                 If certain key is missing from hyperparameters, then `fit()` will not train any models of that type.
                 For example, set `hyperparameters = { 'NN':{...} }` if say you only want to train neural networks and no other types of models.
-            Values = dict of hyperparameter settings for each model type.
-                Each hyperparameter can either be single fixed value or a search space containing many possible values.
+            Values = dict of hyperparameter settings for each model type, or list of dicts.
+                Each hyperparameter can either be a single fixed value or a search space containing many possible values.
                 Unspecified hyperparameters will be set to default values (or default search spaces if `hyperparameter_tune = True`).
                 Caution: Any provided search spaces will be overridden by fixed defaults if `hyperparameter_tune = False`.
+                To train multiple models of a given type, set the value to a list of hyperparameter dictionaries.
+                    For example, `hyperparameters = {'RF': [rf_params1, rf_params2, rf_params3]}` will result in 3 random forest models being trained with separate hyperparameters.
```

Comment:
can we provide mini-example of rf_params1 and rf_params2 (ie single hyperparameter-value)? I don't think 3 are needed here, just 2

Reply:
Added